### PR TITLE
Bug/ch6910 - external initiators do not run "multiply" task

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: core/store/models/common.go [hunk 1]
```diff
@@ -731,6 +732,38 @@ func Merge(inputs ...JSON) (JSON, error) {
...
+func MergeExceptResult(inputs ...JSON) (JSON, error) {
+	output := make(map[string]interface{})
+
+	for _, input := range inputs {
+		switch v := input.Result.Value().(type) {
+		case map[string]interface{}:
+			for key, value := range v {
+				if key == "result" {
+					if _, exists := output["result"]; exists {
+						// Do not overwrite result field
```
Comment (PiotrTrzpil) [inline core/store/models/common.go:747]: minor,  but maybe could print a warning if this happens? i suppose the original field name would need to be changed then. But it's v1, so...

**Review by PiotrTrzpil — APPROVED** [review]
LGTM

File: core/store/models/common.go (hunk 1 above)
Comment (samsondav) [reply core/store/models/common.go:747]: It's quite likely to occur often and not really an error in certain job specs, so I'd say not worth warning here.

**Review by j16r — APPROVED** [review]
How much of our test suite fails if you use this for all merging across all adapters?

**Pull request merged**